doc/source/examining/Loading_Generic_Array_Data.ipynb

```diff
@@ -688,6 +689,48 @@
     "slc.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Species fields"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "One can also supply species fields to a stream dataset, in the form of mass fractions. These will then be used to generate derived fields for mass, number, and nuclei densities of the separate species. The naming conventions for the mass fractions should correspond to the format specified in [the yt documentation for species fields](https://yt-project.org/doc/analyzing/fields.html#species-fields)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "arr = prng.random(size=(64, 64, 64))\n",
+    "data = {\n",
+    "    \"density\": (arr, \"g/cm**3\"),\n",
+    "    \"H_p0_fraction\": (0.37 * np.ones(arr.shape), \"dimensionless\"),\n",
```

Comment:
Minor point, but you could change all of these `np.ones(arr.shape)` to `np.ones_like(arr)` and it may be cleaner.

Reply:
Done